### PR TITLE
git-maintenance: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #3953

### CODE REVIEW

File: pages/common/git-maintenance.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# git-maintenance
+
+> Run tasks to optimize Git repository data.
+> More information: <https://git-scm.com/docs/git-maintenance>.
+
+- Initialize Git config values so any scheduled maintenance will start running on this repository:
```
Comment (bl-ue) [inline pages/common/git-maintenance.md:6]:
```suggestion
- Register the current repository in the user's list of repositories to daily have maintenance run:
```

File: pages/common/git-maintenance.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start running maintenance on the current repository:
+
+`git maintenance start`
+
+- Halt the background maintenance schedule:
+
+`git maintenance stop`
+
+- Remove the current repository from background maintenance:
```
Comment (bl-ue) [inline pages/common/git-maintenance.md:18]:
```suggestion
- Remove the current repository from the user's maintenance repository list:
```

File: pages/common/git-maintenance.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`git maintenance unregister`
+
+- Run a specific maintenance task:
```
Comment (bl-ue) [inline pages/common/git-maintenance.md:22]:
```suggestion
- Run a specific maintenance task on the current repository:
```

File: pages/common/git-maintenance.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Initialize Git config values so any scheduled maintenance will start running on this repository:
+
+`git maintenance register`
+
+- Start running maintenance on the current repository:
+
+`git maintenance start`
+
+- Halt the background maintenance schedule:
```
Comment (bl-ue) [inline pages/common/git-maintenance.md:14]:
```suggestion
- Halt the background maintenance schedule for the current repository:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
See what you think of my comments.

**Comment by Yutyo** [comment]
Seems OK after the requested changes are done.

File: pages/common/git-maintenance.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/git-maintenance.md:6]: I took my time to understand your suggestion 🍺

Comment (bl-ue) [reply pages/common/git-maintenance.md]: Probably could've been better, but I had already spent enough time on it 🙂

**Review by bl-ue — APPROVED** [review]

**Review by Yutyo — APPROVED** [review]
LGTM

**Pull request merged**